tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -932,11 +947,12 @@ public function usersHaveBeenCreated(
 							$userAttributes,
 							__METHOD__ . " userAttributes array does not have key 'userid'"
 						);
-
-						$attributesToCreateUser['email'] = $userAttributes['userid'] . '@owncloud.org';
+						$attributesToCreateUser['email'] = $userAttributes['username'] ?? $userAttributes['userid'] . '@owncloud.org';
```

Comment:
```suggestion
						$attributesToCreateUser['email'] = ($userAttributes['username'] ?? $userAttributes['userid']) . '@owncloud.org';
```

If the email address is not specified, but username is specified, then do you want to append `@owncloud.org` to construct an email address, or is `username` expected to already be a valid email address like `someone@example.org` ?

Reply:
as we are checking that userid exists, we can just rely on it, see 701a4ee35b86eb22fe3a302fc40c98843b142255